python/rikai/pytorch/models/torch.py

```diff
@@ -18,6 +18,7 @@
 from abc import ABC
 from typing import Any, Callable, Optional
 
+import requests
```

Comment:
why `requests` is needed here

Reply:
per pytorch docs, it's fetching the default classification model labels from github

Reply:
ok, I see now. it might be fine for `requests`. Trying to see whether we can limit the exposure of these non-torch / ml essential dependencies. 

`requests` might be just too widely used to hide at this point.

Reply:
yeah I wouldn't be overly concerned about `requests` being included here